marklogic-data-hub/src/main/java/com/marklogic/hub/flow/impl/FlowRunnerImpl.java

```diff
@@ -599,4 +613,66 @@ public void setFlowManager(FlowManager flowManager) {
     public HubConfig getHubConfig() {
         return hubConfig;
     }
+
+    // Brings in artifact settings for running a step
+    protected void applyArtifactSettings(Step step, Map<String, Object> stepConfig) {
```

Comment:
I think we should have unit tests for this, but I'd actually like to write them. I created a subtask for the work.